src/models/sensor-definitions.ts

```diff
@@ -39,7 +40,7 @@ export const SensorStrings:ISensorStrings = {
       "connection_failed_labquest2_alert": "Could not connect to the LabQuest2. Please make sure the address is correct and that the LabQuest2 can be reached from this computer",
       "tare_labquest2_alert": "The LabQuest2 needs to be collecting live data in order to zero. Either set up a new run on the LabQuest2, or click the meter icon in the upper left.",
       "click_here": "click here",
-      "interface_connected": "__interface__ connected",
+      "interface_connected": "__interface__ connected.",
```

Comment:
I guess this makes sense for consistency with other messages, but most of these messages aren't sentences and there are other messages that don't have the period, so I'm not exactly sure what the goal should be here. If there were ever a larger review of our messages, I'd be in favor of removing unnecessary punctuation.

Reply:
When Michael and I did a review of the messages, we found that only some of these are actually shown to the user.  Of the ones that are actually shown, this looked to be the only one that didn't include a period.  Many of these messages (the ones you see without periods) don't appear to be ever shown to the user.  I'm hesitant to remove them in case they were designed for some future use.  However, I at least wanted to unify the one message that deviated from the others within the group that are currently being displayed to the user.